helm/docs/common-configuration.md

```diff
@@ -0,0 +1,450 @@
+# Advanced Configuration
+
+## Client Libraries
+
+The following are the client libraries used in Bold BI by default.
+
+```console
+optionalLibs: 'mongodb,mysql,influxdb,snowflake,oracle,npgsql,google,clickhouse'
+```
+
+Read the optional client library license agreement from the following link.
+
+[Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+
+If you wish to inlcude only specific client libraries note the optional client libraries from the above link as comma separated names in your values.yaml file.
+
+## Persistent Volume
+
+### Name
+
+Persistent volumes were global resources. So if you already have Bold BI installed in your cluster, then the previous persistent volume name will conflict with current installation. Change this name to avoid conflicts with previous Bold BI persistent volumes.
```

Comment:
Persistent volumes were global resources. So if you already have Bold BI installed in your cluster, then the previous persistent volume name will conflict with the current installation. Change this name to avoid conflicts with previous Bold BI persistent volumes.

Reply:
concern addressed